packages/callhierarchy/src/common/glob.ts

```diff
@@ -25,6 +25,7 @@ import * as paths from './paths';
 import { CharCode } from '@theia/core/lib/common/char-code';
 
 /* eslint-disable @typescript-eslint/no-shadow, no-null/no-null */
+
```

Comment:
minor: the following change is unnecessary and can be reverted (it unfortunately adds noise to the history of the file).

Reply:
Fixed